pyintegration/deephaven2/time.py

```diff
@@ -0,0 +1,775 @@
+#
+#   Copyright (c) 2016-2021 Deephaven Data Labs and Patent Pending
+#
+""" This module defines functions for handling Deephaven date/time data. """
+
+import jpy
+
+from deephaven2 import DHError
+from deephaven2.constants import TimeZone
+from deephaven2.dtypes import DateTime, Period
+
+SECOND = 1000000000  #: One second in nanoseconds.
+MINUTE = 60 * SECOND  #: One minute in nanoseconds.
+HOUR = 60 * MINUTE  #: One hour in nanoseconds.
+DAY = 24 * HOUR  #: One day in nanoseconds.
+WEEK = 7 * DAY  #: One week in nanoseconds.
+YEAR = 52 * WEEK  #: One year in nanoseconds.
+
+_JDateTimeUtils = jpy.get_type("io.deephaven.time.DateTimeUtils")
+
+
+def to_datetime(s: str, quiet: bool = False):
```

Comment:
No return type.  Check this on all functions.

Reply:
dtypes.DateTime isn't a Python class but a DType instance. So return type hint isn't possible.